openvdb/math/Math.h

```diff
@@ -351,6 +351,26 @@ inline bool
 isFinite(const Type& x) { return std::isfinite(static_cast<double>(x)); }
 
 
+/// Return @c true if @a x is an infinity value (either positive infinity or negative infinity).
+inline bool
+isInfinite(const float x) { return std::isinf(x); }
+
+/// Return @c true if @a x is an infinity value (either positive infinity or negative infinity).
+template<typename Type, typename std::enable_if<std::is_arithmetic<Type>::value, int>::type = 0>
+inline bool
+isInfinite(const Type& x) { return std::isinf(static_cast<double>(x)); }
+
+
+/// Return @c true if @a x is a NaN (Not-A-Number) value.
+inline bool
+isNan(const float x) { return std::isnan(x); }
```

Comment:
Let's call this "isNaN".

Reply:
I'm not keen on the inconsistency between math::isNaN() and math::Tuple::isNan(), but that's a minor thing. I'll make that change now.

Reply:
You're right, they should be consistent.  But I think I would prefer that we add `Tuple::isNaN` and `Mat::isNaN` and deprecate the `isNan` methods.

Reply:
Do you not think we might annoy users forcing them to change their code when they update simply to capitalize one letter? The current capitalization is at least consistent with Half:

https://github.com/openexr/openexr/blob/master/IlmBase/Half/half.h#L171

Reply:
I changed it back to isNan so this is consistent and users don't have to change their code.